addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -2232,7 +2232,13 @@ var FieldRadio = FieldSelection.extend({
      * @private
      */
     _setValues: function () {
-        if (this.field.type === 'selection') {
+        var selection = this.field.selection;
+        if (this.field.type === 'selection' && this.attrs.visibility) {
+            var restriction = this.attrs.visibility.split(",");
+            this.values = _.filter(selection, function (val) {
+                return _.contains(restriction, val[0]) || val[0] === self.value;
+            });
+        } else if (this.field.type === 'selection') {
```

Comment:
what is it for?

Reply:
task requires showing few options from a `selection` type field that is using `radio` widget for it on view side, that is, to show `Vendor Bill` and `Vendor Credit Note` options on Vendor Bills form view. It's currently not supported, so we introduced `visibility` attribute for `radio` widget(same as we have for `statusbar`) and we used it <a href="https://github.com/odoo/odoo/pull/22158/files?diff=unified#diff-72ec671833aeb837f26b98fcffc8e4aaR249">here</a>

Reply:
fine then.

The implementation is fine, but I'd like to have this feature 
* added by a dedicated commit
* documented properly in https://www.odoo.com/documentation/11.0/reference/javascript_reference.html#relational-fields
* tested